grammar.js

```diff
@@ -139,7 +140,7 @@ module.exports = grammar({
 
     global_statement: ($) => seq(
       maybe_bang($, tokalias($, 'global')),
-      $._separator_first, /[a-zA-A]*/, $._separator,
+      $._separator_first, $.pattern, $._separator,
```

Comment:
Not 100% sure if I should make the change here or in an other PR once this one is merged

Reply:
Making it here is fine I think.